pkg/build/wire/internal/wire/testdata/UnexportedStruct/want/wire_errs.txt

```diff
@@ -1 +1 @@
-example.com/foo/wire.go:x:y: foo not exported by package bar
\ No newline at end of file
+example.com/foo/wire.go:x:y: name foo not exported by package bar
```

Comment:
I added this blindly based on https://drone.grafana.net/grafana/grafana/192278/5/7